evaluation/gorilla/ast_eval_hf.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2023 https://github.com/ShishirPatil/gorilla
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
According to Apache License v2.0 (https://www.apache.org/licenses/LICENSE-2.0):

> You must cause any modified files to carry prominent notices stating that You changed the files.

If I understand correctly, you have made some modifications to their source code. Could you please state it in the header?

Reply:
Added, thanks so much!